### ISSUE TITLE
statistics: show tooltip for median

### ISSUE THREAD

**Issue by pgoellnitz** [globe-swiss/phaenonet-client#40]
Currently only tooltips are shown for the quantiles. Info about the median could be shown on mouseover.

**Comment by pgoellnitz** [comment]
@daria-sl: I think we should also show the median for the statistics. This should be very easy as we already calculate this value. Currently no tooltip is shown when hovering over the median causing the box to flicker, which makes the ux unpleasant.

- We could either show it if the mouse hovers over exactly over the median. However that's just a one pixel wide box and maybe difficult to select. 
- Or we could add the median information to the current popup and also display it when the cursor is over the median.

**Comment by daria-sl** [comment]
"Or we could add the median information to the current popup and also display it when the cursor is over the median." => I'd definitely do this.

Here's a proposition:
![image](https://user-images.githubusercontent.com/65400300/90641644-c08d9300-e231-11ea-811d-8f1760e1a269.png)

Ofc, the pop-up window should get bigger, when the line is added (I can't visualize it perfectly without spending too much time on it...)

Translation in French: **Médiane :**
Translation in Italian: **Mediana:**

**Comment by daria-sl** [comment]
I just had another idea to add into the pop-up: the information about the sample size would be really important for a "serious" interpretation of the boxplots. Could you, in a new line of the pop-up, below the median, also add the following line @pgoellnitz with the respective sample sizes? This would be great =) The lines are given in German, French, Italian:

**Anzahl Beobachtungen:
Nombre d'observations  :
Numero di osservazioni:**

**Comment by daria-sl** [comment]
@pgoellnitz to add these two lines to the pop-up window (Median and Number of Observations) would be in urgent demand by Sebastian, a teacher who wants to make a lesson on how to work with data. What do you think, is it possible to prioritize this functionality right after the intivites part I (which I think are ready now) so that in 3 weeks, this pop-up topic would be online in production? Thank you for your time estimate - if it is not possible, we will find another solution with Sebastian.

**Comment by pgoellnitz** [comment]
Sure, we can do this next - no promises on the timeline. Medians are available right away, number of observations would still need to be calculated.

That said, I still think that this popup is not a way to display a multitude of information. I understand that the visualisation is quite limited but I would rather investigate how all users are working with the aggregated data than base our changes on 'ad-hoc' request of single individuals.

**Comment by daria-sl** [comment]
@pgoellnitz ofc, you have a good point there. 

For the moment: let's stick to medians only due in three weeks. Thank you in advance :)
